tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "resolveJsonModule": true,
     "declaration": true,
     "declarationDir": "types",
-    "jsx": "react-native",
+    "jsx": "react",
```

Comment:
Without this change I'm getting `ReferenceError: Must call super constructor in derived class before accessing 'this' or returning from derived constructor` in tests. If we don't want to make this change could we create a `tsconfig` for testing or provide some sort of transform to sewing-kit?

Reply:
@BPScott

Reply:
Yo! Back in the day we had a thing in our sewing-kit config that force changed the jsx compilation in jest.

It got took out because it didn't seem to be needed. I don't know what in your update stuff has made its return required but it's easy to bring back

https://github.com/Shopify/polaris-react/commit/296dc34bedfbda0f209abe1fb651a2e4127fb97d#diff-1afd2752a047671ab9dea43845beb75a